### DISCUSSION TITLE
New release?

### DISCUSSION THREAD

**Discussion by sreichel** [OpenMage/magento-lts#4275] [Releases]
Thats what we have for now ... 

### Fixes
* Show correct version for main branch by @sreichel in https://github.com/OpenMage/magento-lts/pull/4137
* Allow to use $before parameter in addCss/addJs from XML by @Caprico85 in https://github.com/OpenMage/magento-lts/pull/4151
* Fixed issues #4134 undefined array index label_use_default in Mage_Catalog_Model_Product_Attribute_Backend_Media by @empiricompany in https://github.com/OpenMage/magento-lts/pull/4136
* Fix strlen() null deprecation in AccountController.php by @kiatng in https://github.com/OpenMage/magento-lts/pull/4155
* Added `error` directory to checks + php8.4 fix by @sreichel in https://github.com/OpenMage/magento-lts/pull/4184
* Add email link to set password. See issue #1205. by @kiatng in https://github.com/OpenMage/magento-lts/pull/3262
* Fixed undefined array key "values" in Mage_Catalog_Model_Product_Attribute_Backend_Media by @S0FTWEX in https://github.com/OpenMage/magento-lts/pull/4125
* Fix ACL Resource "admin/sales/order/actions/use" not found by @pbaldauf in https://github.com/OpenMage/magento-lts/pull/4199
* PHP8: Fixes passing null to `str_contains()` by @sreichel in https://github.com/OpenMage/magento-lts/pull/4203
* Fixed missing/wrong admin `_setActiveMenu()` by @sreichel in https://github.com/OpenMage/magento-lts/pull/4209
* Fix wrong Preview link on CMS pages grid by @Caprico85 in https://github.com/OpenMage/magento-lts/pull/4214
* In product list all swatches are always shown even if they are not available when "Display Out of Stock Products" is disabled by @empiricompany in https://github.com/OpenMage/magento-lts/pull/4100
* Openmage admin theme: Change background-color for active sections by @sreichel in https://github.com/OpenMage/magento-lts/pull/4219
* Backport: Added unittest for date and datetime by @sreichel in https://github.com/OpenMage/magento-lts/pull/4210
* Fixed strcmp(): Passing null to parameter #1 ($string1) of type strin… by @kiatng in https://github.com/OpenMage/magento-lts/pull/4212
* Custom product types - increased column length for widgets by @sreichel in https://github.com/OpenMage/magento-lts/pull/4226
* PHP8.1: Added getter method for `Mage_Adminhtml_Block_Widget_Grid_Column::getType()` to return string by @sreichel in https://github.com/OpenMage/magento-lts/pull/4238
* Admin theme: Change background-color for active sections by @sreichel in https://github.com/OpenMage/magento-lts/pull/4246
* Deprecated functionality: preg_match(): Passing null to parameter #4 … by @addison74 in https://github.com/OpenMage/magento-lts/pull/4251
* Bugfix: passing invalid ID to `Mage_Catalog_Model_Url::refreshRewrites()` by @sreichel in https://github.com/OpenMage/magento-lts/pull/4259
* Configurable: remove unused double foreach loop by @Hanmac in https://github.com/OpenMage/magento-lts/pull/4008
* Added `ext-ftp` to requirements by @sreichel in https://github.com/OpenMage/magento-lts/pull/4270
* Fixes configurable swatch color picker in legacy admin theme by @sreichel in https://github.com/OpenMage/magento-lts/pull/4218
* Fixed incorrect datetime in block, ref #1525 by @sreichel in https://github.com/OpenMage/magento-lts/pull/4242
* PHP8: base64_decode(): Passing null is deprecated by @sreichel in https://github.com/OpenMage/magento-lts/pull/4281
* Fix placeholder image in WYSIWYG editor when using a different admin theme by @Caprico85 in https://github.com/OpenMage/magento-lts/pull/4240
* [PHP 8.4] fgetcsv() - The $escape parameter must be provided as its default value will change by @addison74 in https://github.com/OpenMage/magento-lts/pull/4297
* [PHP 8.4] fputcsv() - The $escape parameter must be provided as its default value will change by @addison74 in https://github.com/OpenMage/magento-lts/pull/4298
* Fixes `Mage_Core_Helper_Url::removeRequestParam()` by @sreichel in https://github.com/OpenMage/magento-lts/pull/4295
* PHP84: ignore deprecated E_STRICT by @sreichel in https://github.com/OpenMage/magento-lts/pull/4289
* Fix deprecated message for strlen first value null by @leissbua in https://github.com/OpenMage/magento-lts/pull/4306
* Fixed full name observer calls in config.xml by @sreichel in https://github.com/OpenMage/magento-lts/pull/4309
* Fix `Mage_Adminhtml_Block_Template::isOutputEnabled()` for invalid module by @sreichel in https://github.com/OpenMage/magento-lts/pull/4320
* Fix: cannot add to wishlist from tag page - Customer account by @sreichel in https://github.com/OpenMage/magento-lts/pull/4329
* Fix duplicate block rendering with getSortedChildren() by @justinbeaty in https://github.com/OpenMage/magento-lts/pull/4337
* [PHP 8.x] Deprecated functionality: strtotime(): Passing null to para… by @addison74 in https://github.com/OpenMage/magento-lts/pull/4339
* Fixes line-height for some H4 in admin legacy theme by @sreichel in https://github.com/OpenMage/magento-lts/pull/4335
* Added `ext-pdo_mysql` to checks by @sreichel in https://github.com/OpenMage/magento-lts/pull/4350
* Fix: create empty SimpleXMLElement by @sreichel in https://github.com/OpenMage/magento-lts/pull/4351
* Bugfix: cant create order from admin backend by @sreichel in https://github.com/OpenMage/magento-lts/pull/4348
* Deprecated functionality: preg_replace() by @addison74 in https://github.com/OpenMage/magento-lts/pull/4311
* Fixed null deprecation in Varien/Convert/Parser/Xml/Excel by @kiatng in https://github.com/OpenMage/magento-lts/pull/4375
* Fixed null deprecation in `Mage_Catalog_Model_Product_Option_Type_Text::validateUserValue()` by @kiatng in https://github.com/OpenMage/magento-lts/pull/4357
* Bugfix: `Mage_Adminhtml_Helper_Dashboard_Data::countStores()` on null by @sreichel in https://github.com/OpenMage/magento-lts/pull/4370
* Fix validateHttpVersion to be RFC 1945 and RFC 7230 compliant by @leissbua in https://github.com/OpenMage/magento-lts/pull/4368
* Fix for catalog_product_relation not being deleted when bundle option deleted. by @dannydnz in https://github.com/OpenMage/magento-lts/pull/4395
* Fix unserializeArray on nonserialized string by @Hanmac in https://github.com/OpenMage/magento-lts/pull/4387
* [Tax] add FK to tax/sales_order_tax again by @Hanmac in https://github.com/OpenMage/magento-lts/pull/4334
* Fix Possible Array Access warnings in Captcha Observer by @Hanmac in https://github.com/OpenMage/magento-lts/pull/4424

### Features
* New feature: Added "Newsletter Subscription Form" widget by @empiricompany in https://github.com/OpenMage/magento-lts/pull/4095
* Added support to specify file upload dir in system configuration by @eneiasramos in https://github.com/OpenMage/magento-lts/pull/4079
* Add new method to get config value directly from DB bypassing cache. by @kiatng in https://github.com/OpenMage/magento-lts/pull/4163
* Updated `Varien_Object::getData()` and added `getDataByKey()` & `getDataByPath()` by @sreichel in https://github.com/OpenMage/magento-lts/pull/4205
* Admin config: added input validation & hide disabled options by @sreichel in https://github.com/OpenMage/magento-lts/pull/4232
* Better config for admin RSS links by @sreichel in https://github.com/OpenMage/magento-lts/pull/4243
* Prevent delete/disable CMS pages used in configuration by @sreichel in https://github.com/OpenMage/magento-lts/pull/4239
* Unified admin grid columns and added XML configuration by @sreichel in https://github.com/OpenMage/magento-lts/pull/4225
* Make `Mage_GiftMessage` optional in templates by @Hanmac in https://github.com/OpenMage/magento-lts/pull/4266
* Cache `isModuleOutputEnabled` or `isModuleEnabled` status by @sreichel in https://github.com/OpenMage/magento-lts/pull/4323
* Use `transliterator_transliterate` to generate "url_key" by @sreichel in https://github.com/OpenMage/magento-lts/pull/4315
* New feature: Highlight popular countries in country select by @ma4nn in https://github.com/OpenMage/magento-lts/pull/4089
* Install TinyMCE 7.5 via composer by @sreichel in https://github.com/OpenMage/magento-lts/pull/4113

### Security
* TinyMCE security fix by @sreichel in https://github.com/OpenMage/magento-lts/pull/4157
* Obfuscated login credentials in stacktraces by @sreichel in https://github.com/OpenMage/magento-lts/pull/4388

### DDEV
* [DDEV] Adding n98-magerun support for the test environment by @addison74 in https://github.com/OpenMage/magento-lts/pull/4107
* DDEV: file permission and bugfix by @sreichel in https://github.com/OpenMage/magento-lts/pull/4145
* DDEV: command for local development by @sreichel in https://github.com/OpenMage/magento-lts/pull/4133
* [DDEV] Support for `xhprof`/`xhgui` by @sreichel in https://github.com/OpenMage/magento-lts/pull/4325
* DDEV: updated install script by @sreichel in https://github.com/OpenMage/magento-lts/pull/4407

### Coding style
* PHPCS: fixed php compatibility warning by @sreichel in https://github.com/OpenMage/magento-lts/pull/4148
* PHPCS: ignore `mcrypt` related warnings by @sreichel in https://github.com/OpenMage/magento-lts/pull/4147
* PHP-CS-Fixer: added check for double quotes by @sreichel in https://github.com/OpenMage/magento-lts/pull/4126
* PHPCS: fixed last PHPCompatibility check errors by @sreichel in https://github.com/OpenMage/magento-lts/pull/4164
* Small DocBlock update by @sreichel in https://github.com/OpenMage/magento-lts/pull/4160
* PHP-CS-Fixer: added check for short array syntax and logical operators by @sreichel in https://github.com/OpenMage/magento-lts/pull/4181
* PHP-CS-Fixer: added check for concat spaces by @sreichel in https://github.com/OpenMage/magento-lts/pull/4185
* PHPCS: patched ECG standard by @sreichel in https://github.com/OpenMage/magento-lts/pull/4149
* Rector: CQ - UnusedForeachValueToArrayKeysRector by @sreichel in https://github.com/OpenMage/magento-lts/pull/4111
* Rector: DeadCode - removed useless docblocks by @sreichel in https://github.com/OpenMage/magento-lts/pull/4201
* Rector: added dependency and basic rules by @sreichel in https://github.com/OpenMage/magento-lts/pull/4213
* Skip some rector rules by @sreichel in https://github.com/OpenMage/magento-lts/pull/4227
* Add template files to phpstan checks by @sreichel in https://github.com/OpenMage/magento-lts/pull/4245
* PHPCS-ECG: fail on error by @sreichel in https://github.com/OpenMage/magento-lts/pull/4324
* Code quality: added PHPMD, updated workflow by @sreichel in https://github.com/OpenMage/magento-lts/pull/4303
* Rector: updated UnitTests by @sreichel in https://github.com/OpenMage/magento-lts/pull/4391
* Rector: set methods to private in `Mage.php` by @sreichel in https://github.com/OpenMage/magento-lts/pull/4392
* Rector: removed useless variables by @sreichel in https://github.com/OpenMage/magento-lts/pull/4393
* Follow coding-standard PER-CS2.0 by @sreichel in https://github.com/OpenMage/magento-lts/pull/4372
* PHPCS: merged workflows by @sreichel in https://github.com/OpenMage/magento-lts/pull/4416

### PHPStan
* Phpstan fixes by @sreichel in https://github.com/OpenMage/magento-lts/pull/4116
* PhpStan fixes by @sreichel in https://github.com/OpenMage/magento-lts/pull/4146
* Bump phpstan to level5 by @sreichel in https://github.com/OpenMage/magento-lts/pull/4265
* PhpStan: bumped to 1.12.1 & renamed config files by @sreichel in https://github.com/OpenMage/magento-lts/pull/4183
* PhpStan: L7 update for `Mage_Sendfriend` by @sreichel in https://github.com/OpenMage/magento-lts/pull/4187
* Fixed phpstan on Mage_Customer_Model_Customer by @kiatng in https://github.com/OpenMage/magento-lts/pull/4190
* Some phpStan fixes by @sreichel in https://github.com/OpenMage/magento-lts/pull/4241
* Some phpstan fixes by @sreichel in https://github.com/OpenMage/magento-lts/pull/4273
* Some phpstan fixes by @sreichel in https://github.com/OpenMage/magento-lts/pull/4276
* Some phpstan fixes by @sreichel in https://github.com/OpenMage/magento-lts/pull/4283
* PHPStan: added some class vars to templates by @sreichel in https://github.com/OpenMage/magento-lts/pull/4322
* PHPStan: some fixes by @sreichel in https://github.com/OpenMage/magento-lts/pull/4328
* More phpstan fixes by @sreichel in https://github.com/OpenMage/magento-lts/pull/4291
* PHPStan: some fixes by @sreichel in https://github.com/OpenMage/magento-lts/pull/4346
* PhpStan: some fixes by @sreichel in https://github.com/OpenMage/magento-lts/pull/4363

### PHPUnit
* PhpUnit: use database for tests, added tests by @sreichel in https://github.com/OpenMage/magento-lts/pull/4138
* PHPUnit: added some tests by @sreichel in https://github.com/OpenMage/magento-lts/pull/4204
* Phpunit: ignore timezone in date tests by @sreichel in https://github.com/OpenMage/magento-lts/pull/4284
* Added more PhpUnit tests by @sreichel in https://github.com/OpenMage/magento-lts/pull/4236
* PhpUnit: added tests by @sreichel in https://github.com/OpenMage/magento-lts/pull/4321
* PHPUnit: added tests by @sreichel in https://github.com/OpenMage/magento-lts/pull/4340
* Added test, ref #4352 by @sreichel in https://github.com/OpenMage/magento-lts/pull/4389

### Cleanup
* Removed empty `Mage_Poll` config section by @sreichel in https://github.com/OpenMage/magento-lts/pull/4122
* Removed MTF by @sreichel in https://github.com/OpenMage/magento-lts/pull/4161
* Updated .gitignore, ref #4161 by @sreichel in https://github.com/OpenMage/magento-lts/pull/4167
* Remove `Mage_Sendfriend` by @sreichel in https://github.com/OpenMage/magento-lts/pull/4274
* PHPMD: remove unused private methods/fields by @sreichel in https://github.com/OpenMage/magento-lts/pull/4397
* Cleanup:  removed broken `Mage_Core_Controller_Front_Router` by @sreichel in https://github.com/OpenMage/magento-lts/pull/4356
* Remove gitpod files by @sreichel in https://github.com/OpenMage/magento-lts/pull/4404

### Docs
* Docs: Added section for PhpStorm (SCSS) file watcher by @sreichel in https://github.com/OpenMage/magento-lts/pull/4208
* Documentation v4 📝 - github.io page by @sreichel in https://github.com/OpenMage/magento-lts/pull/4156
* DOCs: update by @sreichel in https://github.com/OpenMage/magento-lts/pull/4349

### Other
* CodeQL: updated workflow by @sreichel in https://github.com/OpenMage/magento-lts/pull/4172
* Added method to make use of `Mage_Core_Model_Security_HtmlEscapedString` easier by @sreichel in https://github.com/OpenMage/magento-lts/pull/4123
* Worklflow: update copyright monthly by @sreichel in https://github.com/OpenMage/magento-lts/pull/4176
* Updated Copyright by @github-actions in https://github.com/OpenMage/magento-lts/pull/4189
* Fixed some typos by @sreichel in https://github.com/OpenMage/magento-lts/pull/4200
* Preview PR in gitpod by @sreichel in https://github.com/OpenMage/magento-lts/pull/4175
* Updated Copyright by @github-actions in https://github.com/OpenMage/magento-lts/pull/4237
* Added polyfill-8.3/8.4 in Composer by @addison74 in https://github.com/OpenMage/magento-lts/pull/4268
* Updated zf1-future to 1.24.1 by @sreichel in https://github.com/OpenMage/magento-lts/pull/4278
* Better caching for phpstan workflow by @sreichel in https://github.com/OpenMage/magento-lts/pull/4269
* Updated Copyright by @github-actions in https://github.com/OpenMage/magento-lts/pull/4327
* Hotfix:  `core_store.store_id` can be `null` by @sreichel in https://github.com/OpenMage/magento-lts/pull/4330
* Bugfix: cant view order in customer account by @sreichel in https://github.com/OpenMage/magento-lts/pull/4345
* Bugfix: hide exception, ref #4303 by @sreichel in https://github.com/OpenMage/magento-lts/pull/4344
* Bugfix: fixed sitemap config depend nodes by @sreichel in https://github.com/OpenMage/magento-lts/pull/4369
* Fixed issue #4384 bug on not able to view order details in frontend. by @kiatng in https://github.com/OpenMage/magento-lts/pull/4385
* Updated Copyright by @github-actions in https://github.com/OpenMage/magento-lts/pull/4400
* Readme: fixed badges by @sreichel in https://github.com/OpenMage/magento-lts/pull/4413

### Dependabot
* Bump squizlabs/php_codesniffer from 3.10.1 to 3.10.2 by @dependabot in https://github.com/OpenMage/magento-lts/pull/4109
* Bump friendsofphp/php-cs-fixer from 3.59.3 to 3.60.0 by @dependabot in https://github.com/OpenMage/magento-lts/pull/4127
* Bump phpstan/phpstan from 1.11.5 to 1.11.7 by @dependabot in https://github.com/OpenMage/magento-lts/pull/4090
* Bump phpseclib/phpseclib from 3.0.39 to 3.0.41 by @dependabot in https://github.com/OpenMage/magento-lts/pull/4154
* Bump friendsofphp/php-cs-fixer from 3.60.0 to 3.62.0 by @dependabot in https://github.com/OpenMage/magento-lts/pull/4153
* Bump friendsofphp/php-cs-fixer from 3.62.0 to 3.64.0 by @dependabot in https://github.com/OpenMage/magento-lts/pull/4178
* Bump phpstan/phpstan from 1.11.7 to 1.12.0 by @dependabot in https://github.com/OpenMage/magento-lts/pull/4177
* Bump tj-actions/changed-files from 44 to 45 by @dependabot in https://github.com/OpenMage/magento-lts/pull/4170
* Bump phpstan/phpstan from 1.12.1 to 1.12.2 by @dependabot in https://github.com/OpenMage/magento-lts/pull/4188
* Bump peter-evans/create-pull-request from 6 to 7 by @dependabot in https://github.com/OpenMage/magento-lts/pull/4193
* Bump symfony/polyfill-php81 from 1.30.0 to 1.31.0 by @dependabot in https://github.com/OpenMage/magento-lts/pull/4195
* Bump symfony/polyfill-php82 from 1.30.0 to 1.31.0 by @dependabot in https://github.com/OpenMage/magento-lts/pull/4194
* Bump symfony/polyfill-php80 from 1.30.0 to 1.31.0 by @dependabot in https://github.com/OpenMage/magento-lts/pull/4197
* Bump symfony/polyfill-php74 from 1.30.0 to 1.31.0 by @dependabot in https://github.com/OpenMage/magento-lts/pull/4198
* Bump phpstan/phpstan from 1.12.2 to 1.12.3 by @dependabot in https://github.com/OpenMage/magento-lts/pull/4196
* Bump phpseclib/phpseclib from 3.0.41 to 3.0.42 by @dependabot in https://github.com/OpenMage/magento-lts/pull/4222
* Bump squizlabs/php_codesniffer from 3.10.2 to 3.10.3 by @dependabot in https://github.com/OpenMage/magento-lts/pull/4221
* Bump composer/composer from 2.7.7 to 2.7.9 by @dependabot in https://github.com/OpenMage/magento-lts/pull/4234
* Bump composer/composer from 2.7.9 to 2.8.1 by @dependabot in https://github.com/OpenMage/magento-lts/pull/4253
* Bump phpstan/phpstan from 1.12.3 to 1.12.6 by @dependabot in https://github.com/OpenMage/magento-lts/pull/4252
* Bump rector/rector from 1.2.5 to 1.2.6 by @dependabot in https://github.com/OpenMage/magento-lts/pull/4254
* Bump rector/rector from 1.2.6 to 1.2.7 by @dependabot in https://github.com/OpenMage/magento-lts/pull/4271
* Bump macopedia/phpstan-magento1 from 1.0.7 to 1.0.8 by @dependabot in https://github.com/OpenMage/magento-lts/pull/4272
* Bump phpstan/phpstan from 1.12.6 to 1.12.7 by @dependabot in https://github.com/OpenMage/magento-lts/pull/4300
* Bump rector/rector from 1.2.7 to 1.2.8 by @dependabot in https://github.com/OpenMage/magento-lts/pull/4301
* Bump shardj/zf1-future from 1.24.1 to 1.24.2 by @dependabot in https://github.com/OpenMage/magento-lts/pull/4319
* Bump pelago/emogrifier from 7.2.0 to 7.3.0 by @dependabot in https://github.com/OpenMage/magento-lts/pull/4332
* Bump composer/composer from 2.8.1 to 2.8.2 by @dependabot in https://github.com/OpenMage/magento-lts/pull/4333
* Bump ezyang/htmlpurifier from 4.17.0 to 4.18.0 by @dependabot in https://github.com/OpenMage/magento-lts/pull/4331
* Bump symfony/process from 5.4.45 to 5.4.46 by @dependabot in https://github.com/OpenMage/magento-lts/pull/4336
* Bump rector/rector from 1.2.8 to 1.2.10 by @dependabot in https://github.com/OpenMage/magento-lts/pull/4343
* Bump phpstan/phpstan from 1.12.7 to 1.12.9 by @dependabot in https://github.com/OpenMage/magento-lts/pull/4342
* Bump composer/composer from 2.8.2 to 2.8.3 by @dependabot in https://github.com/OpenMage/magento-lts/pull/4366
* Bump squizlabs/php_codesniffer from 3.10.3 to 3.11.1 by @dependabot in https://github.com/OpenMage/magento-lts/pull/4367
* Bump macopedia/phpstan-magento1 from 1.0.8 to 1.1.0 by @dependabot in https://github.com/OpenMage/magento-lts/pull/4379
* Bump friendsofphp/php-cs-fixer from 3.64.0 to 3.65.0 by @dependabot in https://github.com/OpenMage/magento-lts/pull/4380
* Bump actions/setup-python from 4 to 5 by @dependabot in https://github.com/OpenMage/magento-lts/pull/4365
* Bump tinymce/tinymce from 7.5.1 to 7.6.0 by @dependabot in https://github.com/OpenMage/magento-lts/pull/4422
* Bump mklkj/tinymce-i18n from 24.11.25 to 24.12.9 by @dependabot in https://github.com/OpenMage/magento-lts/pull/4423
* Bump squizlabs/php_codesniffer from 3.11.1 to 3.11.2 by @dependabot in https://github.com/OpenMage/magento-lts/pull/4420
* Bump phpseclib/phpseclib from 3.0.42 to 3.0.43 by @dependabot in https://github.com/OpenMage/magento-lts/pull/4421
* Bump composer/composer from 2.8.3 to 2.8.4 by @dependabot in https://github.com/OpenMage/magento-lts/pull/4419

## New Contributors
* @github-actions made their first contribution in https://github.com/OpenMage/magento-lts/pull/4189
* @S0FTWEX made their first contribution in https://github.com/OpenMage/magento-lts/pull/4125
* @pbaldauf made their first contribution in https://github.com/OpenMage/magento-lts/pull/4199
* @dannydnz made their first contribution in https://github.com/OpenMage/magento-lts/pull/4395

**Full Changelog**: https://github.com/OpenMage/magento-lts/compare/v20.10.1...v20.11.0

**Comment by addison74** [comment]
It might be time for a new version.

**Comment by S0FTWEX** [comment]
I'm using OpenMage 21.0.0-beta2 in my production (tested before) because it's lighter and faster. So don't forget to deploy also version 21.0.0-beta3 or 21.0.0. ;-)

**Reply by sreichel** [reply]
Can you tell me what's faster in v21?

I think changes made there should have ne notable impact.

**Comment by alexh-swdev** [comment]
How's it going with the release? :)

**Reply by sreichel** [reply]
https://github.com/OpenMage/magento-lts/pull/4113#issuecomment-2477123862

@colinmollenhour and i spent good time to get a opensource-license for TinyMCE v7. **Please do some reviews.**

(@colinmollenhour if there is anything to regarding license-files, let me know. (any reply to you latest email?))

**Reply by colinmollenhour** [reply]
Apologies for the delay! Tiny responded positively to my first round of questions/comments and then this got buried in my inbox... I just sent another response which I think will close all outstanding discussions so that they can send the final agreement and then we should be good.

We do have one technical requirement, though... I will add a comment on #4113

**Reply by sreichel** [reply]
Thanks for working on it. Tech requirement are solved. Tested with git- and composer-install. 

IMHO ... its ready to merge.

What else? Open phpstan-PRs should be merged, so we can upgrade to currently released v2.

- see #4363
- see #4346

**Comment by sreichel** [comment]
I am done for this year.

No more PRs. Only review open issues.

PHPstan v2 will come next year. @justinbeaty, thanks for plugin-update coming first next year.

Release planned before x-mas.

**Comment by sreichel** [comment]
Last call for reviews ... :)

- see https://github.com/OpenMage/magento-lts/pull/4334
- see https://github.com/OpenMage/magento-lts/pull/4424
- see https://github.com/OpenMage/magento-lts/pull/4413
- see https://github.com/OpenMage/magento-lts/pull/4415
- see https://github.com/OpenMage/magento-lts/pull/4416

Waiting for 

- see https://github.com/OpenMage/magento-lts/pull/4418

**Comment by colinmollenhour** [comment]
I updated the release page description with your description from this discussion.

**Comment by sreichel** [comment]
4415 can added  later ... we only need to bump the version number.

If you are ok with it, hit the "button"

https://www.youtube.com/watch?v=FlFt_W4664M

**Comment by colinmollenhour** [comment]
I wasn't holding off for 4415. The only thing that is a blocker is the TinyMCE agreement...

**Comment by sreichel** [comment]
Composer-plugin also supports v6.8 .... we cann downgrade any time. Go for it?

**Reply by colinmollenhour** [reply]
If you want to get it final this week that's probably the best way.

**Reply by sreichel** [reply]
After the time we spent for it .... 

... id prefer v7 and downgrade if Tiny changed their mind.

**Reply by colinmollenhour** [reply]
Ok so you think the current 20.11.0 tag is good to make the current latest release now?

**Reply by sreichel** [reply]
Yes. I do.

**Reply by colinmollenhour** [reply]
Published as latest. Thanks for all the work you put into it, Sven!

**Reply by sreichel** [reply]
Thanks for your help with TinyMCE!

Beside all that php84 fixes and features it is my personal highlight in this release.

**Reply by colinmollenhour** [reply]
Let me know if you'd like to add a "Highlights" section

**Reply by sreichel** [reply]
Highlight section for release notes?

I think i can do it on my own.  I'll update DOCs asap ...
